geometry_msgs/PoseWithCovariance pose
geometry_msgs/PoseWithCovariance static_pose